openpype/hosts/fusion/hooks/pre_fusion_setup.py

```diff
@@ -14,21 +17,108 @@ class FusionPrelaunch(PreLaunchHook):
     as set in openpype/hosts/fusion/deploy/fusion_shared.prefs to enable
     the OpenPype menu and force Python 3 over Python 2.
 
+    VERSION variable is used because from the Fusion v16 the profile folder
+    is project-specific, but then it was abandoned by devs,
+    and despite it is already Fusion version 18, still FUSION16_PROFILE_DIR
+    is used. The variable is added in case the version number will be
+    updated or deleted so we could easily change the version or disable it.
```

Comment:
I feel like this comment should be in the docstring but directly above the `VERSION` variable. What do others think?